ui/store/actions.ts

```diff
@@ -3003,6 +3003,21 @@ export function setUseNftDetection(
   };
 }
 
+export function setUse4ByteResolution(
```

Comment:
Can you add tests for this?

Reply:
for an example see [here](https://github.com/MetaMask/metamask-extension/pull/16746/files#diff-4b37b6118f4687b9322f4f4a201835d3744df8ce7cf0f8ee63122ea33b3d3343R1562)

Reply:
we can also add a test on `ui/pages/settings/security-tab/security-tab.test.js` ([see here](https://github.com/MetaMask/metamask-extension/pull/16746/files#diff-e9d0c76efec92dedf8825df0065ee0d1ac2b0cd2df99e4a1d4f7cfa499fc9c2cR87) -- although this has now been much simplified by the use of the `toggleCheckbox()` testing method in that file)